sr_port/handle_defered_signal.c

```diff
@@ -0,0 +1,56 @@
+/****************************************************************
```

Comment:
The name of this routine (handle_defered_signal() - which by the way should be deferred I think instead - with two 'r's to match the routine name) is a bit close to the deferred_signal_handler() routine that already exists. They are doing different things but have too-similar names.

Reply:
Do you have any suggestions on a better name?

Reply:
Perhaps rename both routines but rename them to deferred_signal_handler_exit (for the existing routine) and deferred_signal_handler_noexit (for this new routine)? That seems to be the main difference between them.

Reply:
Thanks. How about a refinement of that some more. deferred_exit_handler and deferred_signal_handler.

Reply:
This is addressed in https://github.com/YottaDB/YottaDB/pull/249

Reply:
That sounds good to me.